runtime/sema/check_member_expression.go

```diff
@@ -373,33 +373,12 @@ func (checker *Checker) visitMember(expression *ast.MemberExpression) (accessedT
 
 // isReadableMember returns true if the given member can be read from
 // in the current location of the checker, along with the authorzation with which the result can be used
-func (checker *Checker) isReadableMember(accessedType Type, member *Member, accessRange func() ast.Range) (bool, Access) {
-	var mapAccess func(EntitlementMapAccess, Type) (bool, Access)
-	mapAccess = func(mappedAccess EntitlementMapAccess, accessedType Type) (bool, Access) {
```

Comment:
This function didn't need to be a closure, so extracted it to a top-level function